src/js/components/TabPanesComponent.jsx

```diff
@@ -0,0 +1,65 @@
+var React = require("react/addons");
+
+var AppListComponent = require("../components/AppListComponent");
+var DeploymentsListComponent =
+  require("../components/DeploymentsListComponent");
+var TabPaneComponent = require("../components/TabPaneComponent");
+var TogglableTabsComponent = require("../components/TogglableTabsComponent");
+
+var tabs = [
+  {id: "/apps", text: "Apps"},
+  {id: "/deployments", text: "Deployments"}
+];
+
+var TabPanesComponent = React.createClass({
+  displayName: "TabPanesComponent",
+
+  contextTypes: {
+    router: React.PropTypes.func
+  },
+
+  getInitialState: function () {
+    return {
+      tabId: this.getTabId()
+    };
+  },
+
+  componentWillReceiveProps: function () {
+    this.setState({
+      tabId: this.getTabId()
+    });
+  },
+
+  getTabId: function () {
+    var path = this.context.router.getCurrentPathname();
+
+    if (tabs.find(function (tab) {
```

Comment:
So I can confirm that by doing:

`npm install --save-dev babel`

and then adding  `require("babel/polyfill");`  to our `main.jsx` solves the issue. I'm in favor of the ES6 iterators, so I suggest doing that in a commit!